prbt_support/urdf/simple_gripper_brackets.urdf.xacro

```diff
@@ -17,7 +17,19 @@ limitations under the License.
 -->
 
 <robot xmlns:xacro="http://www.ros.org/wiki/xacro">
-  <xacro:macro name="simple_gripper_brackets" params="gripper_name parent_left parent_right">
+  <xacro:macro name="simple_gripper_brackets"
+               params="gripper_name
```

Comment:
You could pass gripper_name as default, too: `gripper_name:=^`

Reply:
I agree, but the pipe is not necessary? So you would just have the additional option to use `gripper_name` from the outer scope. Otherwise it has to be specified.

Reply:
Ok, I see. Implemented, although I'm not a fan of implicit parameters...